wallet/datastore.go

```diff
@@ -222,11 +246,12 @@ func (pg *Postgres) InsertWallet(wallet *walletutils.Info) error {
 
 // SetAnonymousAddress sets the anon addresses of the provided wallets
 func (pg *Postgres) SetAnonymousAddress(ID string, anonymousAddress *uuid.UUID) error {
```

Comment:
can we just delete `SetAnonymousAddress` now? I thought we were only using it in `wallet/service.go`

Reply:
let's open up a follow up issue for this